bin/fypp

```diff
@@ -2574,6 +2590,16 @@ class Fypp:
         return None
 
 
+    def process_dependencies(self, outfile, depfile):
```

Comment:
Let's call this rather `write_dependencies()`, as this is what it actually does.